### DISCUSSION TITLE
GitHub removed the repo explorer from the top right of the homescreen?

### DISCUSSION THREAD

**Discussion by benjysugi** [community/community#14968] [Other Feature Feedback, Questions, & Ideas]
Is it just me or has github removed the section that used to have 3 recommended repos that was on the top right of the homescreen and now it's just a blank empty space

I'm not really happy with the new change since that was quite a big feature for me

**Comment by 0x3C50** [comment]
Same, they also entirely removed repositories from the explore section. Not a big fan

**Comment by ghost** [comment]
Same here.

**Comment by llama-for3ver** [comment]
me rn: 😭

**Comment by brotherbeer** [comment]
Oh no~ my favorite feature is gone

**Comment by mrjones2014** [comment]
If you're trying to force us into using the "For You" algorithmic tab, it's not going to work.

**Comment by RundesBalli** [comment]
We are lucky: it is back

**Reply by benjysugi** [reply]
yup
